src/modules/auth/index.js

```diff
@@ -22,6 +23,14 @@ export default class AuthModule extends Module {
 		this.App.addSplashScreenRequestor(this);
 
 		this.Authorization = this.Config.get("Authorization"); // Get Authorization settings from configuration
+
+		// Access control screen
+		app.Router.addRoute({
+			path: '/access-control',
```

Comment:
I would really like to have a path prefix here to avoid conflicts with the rest of the application space.

`/auth/access-control`

Reply:
Implemented